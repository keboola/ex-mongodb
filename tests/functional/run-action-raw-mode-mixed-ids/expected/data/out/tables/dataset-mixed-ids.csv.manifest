{"primary_key":[],"incremental":false}
