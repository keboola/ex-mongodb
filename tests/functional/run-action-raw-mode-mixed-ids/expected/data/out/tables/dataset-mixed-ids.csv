"id","data"
"123","{""_id"":123}"
"123.456","{""_id"":123.456}"
"123456","{""_id"":""123456""}"
"94fa7ddd","{""_id"":""94fa7ddd""}"
"94fa8181","{""_id"":""94fa8181""}"
"94fa8213","{""_id"":""94fa8213""}"
"","{""_id"":{""key"":""value""}}"
"5716054bee6e764c94fa7ddd","{""_id"":{""$oid"":""5716054bee6e764c94fa7ddd""}}"
"5716054bee6e764c94fa8181","{""_id"":{""$oid"":""5716054bee6e764c94fa8181""}}"
